products/claim-appeal-status/document-status/engineering/drop-offs-after-failures-spike.md

```diff
@@ -0,0 +1,244 @@
+# Spike Writeup: Tracking Drop-offs After Failures
+
+**Issue**: [#120624](https://github.com/department-of-veterans-affairs/va.gov-team/issues/120624)
+**Epic**: [#120479](https://github.com/department-of-veterans-affairs/va.gov-team/issues/120479) – Document Status Refinements and Clarifications
+**Team**: BMT Team 2 (Bee's Knees)
+**Author**: @dfitchett
+**Status**: Draft
+**Date**: 11/12/2025
+
+---
+
+## 1. Executive Summary
+
+### Objective
+This spike explores options for tracking whether Veterans exit the Claim Status Tool (CST) after encountering a document submission failure, and investigates whether such exits indicate abandonment or expected offline resubmission behavior. The goal is to assess whether Google Analytics funnel analysis or another approach can meaningfully distinguish between these two very different user behaviors.
+
+### Key Findings
+GA Funnel exploration could be used to determine actions taken after the user has a failed upload (Type 1) and after the user sees
+their document that failed to get to VBMS via Lighthouse (Type 2).
+
+#### Recommended Events
+Currently the presence of Type 2 errors do not fire off events, however it could be interesting to record an event when the user navigates to CST and sees it. This would require adding a new event (maybe named `claims-upload-silent-failure`) that includes the number of failed documents. Then implement the second funnel below.
+
+#### Recommended Funnels
+Type 1 Drop-off (navigation away from page)
+1. User Navigates to CST Files page or Evidence Request Page
+1. User uploads file(s) (fires off `claims-upload-start` event)
+1. One or more file(s) fail to upload showing Type 1 Unknown Error message (fires off `claims-upload-failure` event)
+1. User navigates away from CST Files page or Evidence Request Page
+
+Type 2 Drop-off (navigation away from CST - Status Tab or Files Tab)
```

Comment:
does navigating between CST pages like Status to FIles count as drop-off or only leaving CST entirely?